### ISSUE TITLE
Apps will not install from community installer

### ISSUE THREAD

**Issue by mikeb3408** [tonesto7/homebridge-smartthings-v2#162]
Hello!

I have the community installer loaded on my SmartThings IDE and connected to your repository and published. 

When I open the smarthings app and login with my Samsung account I see the list of apps available. However when I click any of them all it does is take me back to the top of the page of apps. I also see that every app does as being downloaded zero times but on all of the tutorials those show a much higher number. 

Any ideas?

**Comment by Jsunm1** [comment]
I also have this issue.
We were told to try the regular SmartThings App and add it since maybe something was updated.

However it looks like it starts to parse something and then times out saying, "We're having trouble connecting.
Check your network connection and try again. <OK> "

**Comment by stckhlm73** [comment]
Yes, I have exactly the same issue too. To try and rule out a local issue I've tried multiple devices and browsers but to no avail! Please help!

**Comment by Thomas2702** [comment]
Was having the same issue, used the method in this video and it worked perfect.

https://www.youtube.com/watch?v=ORyEn5uVJ1k&feature=youtu.be&fbclid=IwAR0fRkulwJOreuUaI1ly4g6YMCC64Swww2vsse1KdaYHagMHT1_3RldST74

**Comment by Jsunm1** [comment]
That videos approach worked like a charm!  Thank You!

**Comment by stckhlm73** [comment]
Solved! Thanks, @Thomas2702!
